### PR TITLE
add jadebanares

### PR DESCRIPTION
**Pull request by jade-arinal-banares**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1939"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: public/data/jadebanares.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "avatar": "http://github.com/jade-arinal-banares.png",
+  "links": [
+    {
+      "name": "Check out my Portfolio",
+      "url": "https://jade-arinal-banares.vercel.app/",
+      "icon": "globe"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/notifications/",
```
Comment (Vyvy-vi) [inline public/data/jadebanares.json:14]: This link goes to the default LinkedIn feed. Perhaps you can add your profile link here?

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]

**Review by accodes21 — CHANGES_REQUESTED** [review]
Change the LinkedIn link.
Rest all is good.

File: public/data/jadebanares.json (hunk 1 above)
Comment (jade-arinal-banares) [reply public/data/jadebanares.json:14]: Oh I'm sorry, I forgot to change that. Will do :))

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**